images/openshift-enterprise-cli.yml

```diff
@@ -1,16 +1,14 @@
 content:
   source:
     alias: ose
-    modifications:
-    - action: replace
-      match: origin-clients
-      replacement: atomic-openshift-clients
-    path: images/cli
+    dockerfile: images/cli/Dockerfile.rhel
 distgit:
   namespace: containers
 enabled_repos:
 - rhel-server-ose-rpms
 from:
+  builder:
+  - image: openshift/golang-builder:1.10
```

Comment:
Note: Nothing to change, but soon this will become a stream so we can change it in one place.

Reply:
Good

Reply:
The change has already gone through. I suggest updating this now.
Just needs to look like:
```
from:
  builder:
  - stream: golang
  stream: rhel
```